### PR TITLE
Editing pass for the bigtest platform

### PR DESCRIPTION
**Pull request by jenweber**
@minkimcello here's the editing pass on the BigTest platform docs. I wanted to make a PR so that you could review the changes. Go ahead and make any edits you want on this branch and merge when you are ready. Thanks!

### CODE REVIEW

File: website/docs/docs/platform/2-writing-your-first-test.md [hunk 1]
```diff
@@ -40,7 +44,41 @@ export default test('bigtest todomvc')
...
+```
+
+## Interactors
+
+In the examples above, we used Interactors like `Button` and `Heading` to locate something in the UI, interact with it like a user would, and make a test assertion. Learn more about how interactors work in our [Interactors](../interactors) guides.
+
+## Test order
+
+If you are familiar with other testing libraries, might have a few questions at this point about testing order. Here is some basic information to help you connect the dots for how BigTest works.
+
```
Comment (jorgelainfiesta) [inline website/docs/docs/platform/2-writing-your-first-test.md:78]: I really like how this is introduced. Can we add some more simple examples? We haven't seen `children` yet for instance.

**Review by jorgelainfiesta — COMMENTED** [review]
Looking great y'all!

File: website/docs/docs/platform/2-writing-your-first-test.md (hunk 1 above)
Comment (minkimcello) [reply website/docs/docs/platform/2-writing-your-first-test.md:78]: the children example is in between lines 35-44

**Review by minkimcello — APPROVED** [review]

**Comment by minkimcello** [comment]
This is fantastic. Thank you. I'll go ahead and merge.

**Pull request merged**